look-and-feel/css/src/Button/Button.scss

```diff
@@ -19,15 +21,22 @@
   cursor: pointer;
   user-select: none;
 
+  & > svg {
+    aspect-ratio: 1;
+    fill: var(--content-color);
+  }
```

Comment:
y a ptetre un truc que je rate, pourquoi fill ne vaut pas `currentcolor` ?

Reply:
tout simplement car je ne connaissait pas 😆 
je vais tester, ce sera en effet sûrement plus simple

Reply:
Fix in https://github.com/AxaFrance/design-system/pull/665/commits/c5b2414e5a416a3265f834c5b486cf2e0ed10660